diff --git a/src/main/application/searchdefinitions/music.sd b/src/main/application/searchdefinitions/music.sd
index d6bcaac..158c217 100644
--- a/src/main/application/searchdefinitions/music.sd
+++ b/src/main/application/searchdefinitions/music.sd
@@ -16,7 +16,7 @@ search music {
         }
 
         field duration type int {
-            indexing: summary
+            indexing: summary | attribute
         }
 
 	    field year type int {
